packages/terra-button/CHANGELOG.md

```diff
@@ -3,6 +3,8 @@ Changelog
 
 Unreleased
 ----------
+### Added
```

Comment:
I think this should be in Menu's changelog? 😄

Reply:
Geeze, my brain is just gone today.